c2corg_ui/static/js/addassociationdirective.js

```diff
@@ -55,7 +55,13 @@ app.AddAssociationController = function(appApi, $attrs) {
   this.api_ = appApi;
 
 
-  this.dataset = $attrs['dataset'];
+  /**
+   * @private
+   * @type {string}
+   */
+  this.dataset_ = $attrs['dataset'];
```

Comment:
@gberaudo For ``this.addedDocuments = [];`` a few line below, do we really need to specify an initial value (``[]``)?

For ``this.parentId;`` I think the ``// bound from directive`` comment is not needed (or we could add it at many other places...).